<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uiUserProfile</class>
 <widget class="QDialog" name="uiUserProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>435</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>435</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>360</width>
    <height>435</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Profile</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/images/logo_64.png</normaloff>:/images/logo_64.png</iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>435</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <item>
     <widget class="QLabel" name="label_nick">
      <property name="text">
       <string>Nick</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_avatar">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string>Avatar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit_shortDesc">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>70</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>70</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::FieldsStayAtSizeHint</enum>
      </property>
      <property name="labelAlignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
      <property name="formAlignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <item row="4" column="0">
       <widget class="QLabel" name="label_city">
        <property name="text">
         <string>City:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="lineEdit_city">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_birthdate">
        <property name="text">
         <string>Birthdate:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEdit_birthdate">
        <property name="minimumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_country">
        <property name="text">
         <string>Country:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="lineEdit_country">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="lineEdit_type">
        <property name="minimumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_www">
        <property name="text">
         <string>Website:</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QLineEdit" name="lineEdit_www">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_email">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QLineEdit" name="lineEdit_email">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_sex">
        <property name="text">
         <string>Sex:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_sex">
        <property name="minimumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>114</width>
          <height>25</height>
         </size>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_longDesc">
        <property name="text">
         <string>Desc:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPlainTextEdit" name="plainTextEdit_longDesc">
        <property name="minimumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>50</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="standardButtons">
       <set>QDialogButtonBox::Close</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
